### PR TITLE
Fix typos and displays issues

### PR DESCRIPTION
**Pull request by nqb**

### CODE REVIEW

**Comment by nqb** [comment]
BUMP

File: source/advanced-topics.adoc [hunk 1]
```diff
@@ -136,10 +136,10 @@ The "Generating signature" message appears in both the binary and source packagi
 
 [NOTE]
 --
-When using the `--sign` option for `rpmbuild`, use only -bb or -ba options for package building.
+When using the `--sign` option for `rpmbuild`, use only `-bb` or `-ba` options for package building. `-ba` option mean build binary *and* source packages.
 --
 
-To verify the signature of a package, use the rpm command with `--checksig` option. For example:
+To verify the signature of a package, use the `rpmbuild` command with `--checksig` option. For example:
```
Comment (xsuchy) [inline source/advanced-topics.adoc]: This should be `rpm` not `rpmbuild`.

**Comment by xsuchy** [comment]
Besides that one issue, it is good. Fix it please and I will merge it.

**Comment by nqb** [comment]
@xsuchy: done !

**Pull request merged**